.circleci/config.yml

```diff
@@ -0,0 +1,110 @@
+version: 2.1
+executors:
+  default:
+    working_directory: ~/mattermost-webapp
+    docker:
+      - image: mattermost/mattermost-build-webapp:oct-2-2018
+
+jobs:
+  install:
+    executor:
+      name: default
+    steps:
+      - checkout
+      - restore_cache:
```

Comment:
Could we leverage aliases to avoid duplicating the cache configuration? See https://github.com/mattermost/mattermost-cloud/blob/master/.circleci/config.yml#L7 for an example.

Reply:
cool, much cleaner :) thanks for that